models/neighborhoods.ts

```diff
@@ -0,0 +1,15 @@
+export type Neighborhood = {
+  'Neighborhood Name': string
+  'NTA Name': string
+  BoroName: string
+  CountyFIPS: string
+  NTACode: string
+  State: string
+  Address: string
+  GeoCode: string
+  /** List of group IDs */
+  'Neighborhood Groups Providing Service is In'?: string[]
```

Comment:
wow is that the actual name for the property on AirTable?

Reply:
Apparently lool